SUMMARY.md

```diff
@@ -2,4 +2,9 @@
 
 * [Introduction](README.md)
 * [Front_End](./front-docs/LANDING_PAGE.md)
-* [Main_Page](./front-docs/MAIN_PAGE.md) 
+* [Main_Page](./front-docs/MAIN_PAGE.md)
```

Comment:
I think that the underscores are a little ugly xD

Reply:
Those underscores are NOT mine. They where there when I clone the repo.